src/main/java/io/th0rgal/oraxen/commands/PrintFontCommand.java

```diff
@@ -0,0 +1,111 @@
+                if (i % 3 == 0) {
+                    sender.spigot().sendMessage(cb.create());
+                    cb = new ComponentBuilder();
+                }
+                i++;
+            }
+            if (cb.getParts().size() != 0)
+                sender.spigot().sendMessage(cb.create());
+            return;
+        }
+        Glyph g = OraxenPlugin.get().getFontManager().getGlyphs().stream().filter(glyph -> glyph.getName().equals(glyphName)).findAny().orElse(null);
+        sender.spigot().sendMessage(printClickableMsg(ChatColor.WHITE + g.getName(), String.valueOf(g.getCharacter()), "§r" + g.getCharacter()));
+    }
+
+    /**
+     * Parses code input to print a unicode list with decimal version of UTF-16
+     * the format is {hex unicode id}+{range to display} like this: "E000+10"
+     * @param sender command sender
+     * @param code unicode symbol with formatted range
+     */
+    private void printUnicode(CommandSender sender, String code) {
+        try {
+            char utf;
+            int range = 1;
+            if (code.matches("[A-Za-z0-9]{4}\\+[0-9]+")) {
+                String[] splitted = code.split("\\+");
+                utf = new String(Hex.decodeHex(splitted[0].toCharArray()), StandardCharsets.UTF_16BE).toCharArray()[0];
+                range = Integer.parseInt(splitted[1]);
+            } else {
+                Bukkit.getLogger().info(code);
+                utf = new String(Hex.decodeHex(code.toCharArray()), StandardCharsets.UTF_16BE).toCharArray()[0];
+            }
+            ComponentBuilder componentBuilder = new ComponentBuilder();
+            for (int i = 0; i < range; i++) {
+                componentBuilder.append(printClickableMsg(ChatColor.WHITE + "[" + ChatColor.AQUA + "U+" + Integer.toHexString(utf).toUpperCase() + "," + ((int) utf) + "(dec)" + ChatColor.WHITE + "] ",
+                        String.valueOf(utf),
+                        "§r" + utf));
+                if (i == 2) {
+                    sender.spigot().sendMessage(componentBuilder.create());
+                    componentBuilder = new ComponentBuilder();
+                }
+                utf++;
+            }
+            if (componentBuilder.getParts().size() != 0)
+                sender.spigot().sendMessage(componentBuilder.create());
+
+        } catch (DecoderException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private void printHelpTitle(CommandSender sender) {
+        sender.sendMessage(ChatColor.RED + "The following text is " + ChatColor.BOLD + ChatColor.DARK_RED + "CLICKABLE and HOVERABLE" + ChatColor.RESET + ChatColor.RED + "!");
+    }
+
+    private BaseComponent[] printClickableMsg(String text, String unicodeChar, String hoverText) {
+        BaseComponent[] components = new ComponentBuilder(text)
+                .event(new ClickEvent(ClickEvent.Action.SUGGEST_COMMAND, unicodeChar))
```

Comment:
Seems better to simply use COPY_TO_CLIPBOARD action here and then add a hovermsg or helpmessage to reflect that

Reply:
the hover is for displaying unicode chars.
since the char may be very large, writing anything in hover text may be hidden by the image

Reply:
You can just change the `This message is CLICKABLE` message then.
Make it say "Click one of the below to copy the unicode" or something